src/providers/profile/profile.ts

```diff
@@ -674,8 +674,14 @@ export class ProfileProvider {
 
   private askToEncryptKey(key): Promise<any> {
     if (!key) return Promise.resolve();
+    // if the key is already encrypted, keep it that way for new wallets
     if (key.isPrivKeyEncrypted()) return Promise.resolve();
 
+    // do not request encryption if wallets were already created without it
+    const wallets = this.getWallets({ keyId: key.id });
```

Comment:
At this point, you have already created the wallets.

Reply:
So, it will be true

Reply:
bindWalletClient function is called after the key encryption

Reply:
Ok, you're right.